app/src/main/res/values/colors.xml

```diff
@@ -3,5 +3,5 @@
     <color name="colorPrimary">#3F51B5</color>
     <color name="colorPrimaryDark">#303F9F</color>
     <color name="colorAccent">#FF4081</color>
-    <color name="contest_app_textcolor">#898989</color>
+    <color name="contestAppTextColor">#898989</color>
```

Comment:
I don't see this being used, in which case you could remove it for now. Otherwise, maybe this could be the new value of `colorPrimary` instead of a new resource, if it will be app-wide.